tests/network/l2_bridge/test_bridge_nic_hot_plug.py

```diff
@@ -448,6 +449,22 @@ def test_vmi_spec_updated_with_hot_plugged_interface(
             interface_name=hot_plugged_interface.name,
         )
 
+    @pytest.mark.polarion("CNV-12223")
+    @pytest.mark.dependency(
+        name="test_no_restart_needed_status_in_vm_after_hot_plug",
```

Comment:
why is the name needed here? no test depends on this one an di think it is confusing to someone reading the code as it leads to an assumption that some test does depend on it